drivers/misc/mcux_flexio/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2024, STRIM, ALC
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+
+zephyr_library_sources(mcux_flexio.c)
```

Comment:
Can we add `zephyr_include_directories(.)` to this file. That way when including the header file, we don't have to include the complete path.

Reply:
I don't think that's the way to do this; the header should go in the include directory in the first place if it is intended to be a public header

Reply:
That sounds like a better suggestion. We could move the .h file to the include folder.

Reply:
I will do it.